### PR TITLE
chore: Fix readme linter errors for input plugins A-D

### PR DESCRIPTION
**Pull request by reimda**
Use the linter in https://github.com/influxdata/telegraf/pull/10916 to clean up input plugin A-D readme markdown files.

The linter has more rules for input plugins, including rules that require sections for example output and metric format. I've fixed formatting problems, but quite a few of the plugins don't have content for the required sections. They will require some research, so there is more work to do and I've marked this PR as a draft.

Here is the linter output. It shows which plugins lack which sections.

```shell
Pass plugins/inputs/activemq/README.md
Pass plugins/inputs/aerospike/README.md
Pass plugins/inputs/aliyuncms/README.md
Pass plugins/inputs/amd_rocm_smi/README.md
plugins/inputs/amqp_consumer/README.md:1: missing required section 'Example Output'
plugins/inputs/amqp_consumer/README.md:1: missing required section 'Metrics'
Fail plugins/inputs/amqp_consumer/README.md, 2 failed assertions
Pass plugins/inputs/apache/README.md
Pass plugins/inputs/apcupsd/README.md
Pass plugins/inputs/aurora/README.md
Pass plugins/inputs/bcache/README.md
Pass plugins/inputs/beanstalkd/README.md
Pass plugins/inputs/beat/README.md
Pass plugins/inputs/bind/README.md
Pass plugins/inputs/bond/README.md
plugins/inputs/burrow/README.md:1: missing required section 'Example Output'
Fail plugins/inputs/burrow/README.md, 1 failed assertions
Pass plugins/inputs/cassandra/README.md
Pass plugins/inputs/ceph/README.md
plugins/inputs/cgroup/README.md:1: missing required section 'Example Output'
Fail plugins/inputs/cgroup/README.md, 1 failed assertions
Pass plugins/inputs/chrony/README.md
plugins/inputs/cisco_telemetry_mdt/README.md:1: missing required section 'Metrics'
Fail plugins/inputs/cisco_telemetry_mdt/README.md, 1 failed assertions
Pass plugins/inputs/clickhouse/README.md
plugins/inputs/cloud_pubsub/README.md:1: missing required section 'Example Output'
plugins/inputs/cloud_pubsub/README.md:1: missing required section 'Metrics'
Fail plugins/inputs/cloud_pubsub/README.md, 2 failed assertions
plugins/inputs/cloud_pubsub_push/README.md:1: missing required section 'Example Output'
plugins/inputs/cloud_pubsub_push/README.md:1: missing required section 'Metrics'
Fail plugins/inputs/cloud_pubsub_push/README.md, 2 failed assertions
Pass plugins/inputs/cloudwatch/README.md
Pass plugins/inputs/conntrack/README.md
Pass plugins/inputs/consul/README.md
plugins/inputs/consul_agent/README.md:1: missing required section 'Example Output'
Fail plugins/inputs/consul_agent/README.md, 1 failed assertions
Pass plugins/inputs/couchbase/README.md
Pass plugins/inputs/couchdb/README.md
plugins/inputs/csgo/README.md:1: missing required section 'Example Output'
Fail plugins/inputs/csgo/README.md, 1 failed assertions
Pass plugins/inputs/dcos/README.md
plugins/inputs/directory_monitor/README.md:1: missing required section 'Metrics'
plugins/inputs/directory_monitor/README.md:1: missing required section 'Example Output'
Fail plugins/inputs/directory_monitor/README.md, 2 failed assertions
Pass plugins/inputs/disk/README.md
Pass plugins/inputs/diskio/README.md
plugins/inputs/disque/README.md:1: missing required section 'Example Output'
Fail plugins/inputs/disque/README.md, 1 failed assertions
Pass plugins/inputs/dmcache/README.md
Pass plugins/inputs/dns_query/README.md
Pass plugins/inputs/docker/README.md
Pass plugins/inputs/docker_log/README.md
Pass plugins/inputs/dovecot/README.md
Pass plugins/inputs/dpdk/README.md
```

### CODE REVIEW

File: plugins/inputs/cloudwatch/README.md [hunk 1]
```diff
@@ -4,60 +4,50 @@ This plugin will pull Metric Statistics from Amazon CloudWatch.
...
-
-  ## Endpoint to make request against, the correct endpoint is automatically
-  ## determined and this option should only be set if you wish to override the
-  ## default.
-  ##   ex: endpoint_url = "http://localhost:8000"
-  # endpoint_url = ""
-
-  ## Set http_proxy (telegraf uses the system wide proxy settings if it's is not set)
-  # http_proxy_url = "http://localhost:8888"
+[[inputs.cloudwatch]] ## Amazon Region region = "us-east-1"
```
Comment (powersj) [inline plugins/inputs/cloudwatch/README.md:25]: do these changes to the sample config need to happen in the config.conf as well?

File: plugins/inputs/cloudwatch/README.md [hunk 2]
```diff
@@ -4,60 +4,50 @@ This plugin will pull Metric Statistics from Amazon CloudWatch.
...
+
+  ## Amazon Credentials ## Credentials are loaded in the following order ## 1)
+  Web identity provider credentials via STS if role_arn and
+  web_identity_token_file are specified ## 2) Assumed credentials via STS if
+  role_arn is specified ## 3) explicit credentials from 'access_key' and
+  'secret_key' ## 4) shared profile from 'profile' ## 5) environment variables
+  ## 6) shared credentials file ## 7) EC2 Instance Profile # access_key = "" #
+  secret_key = "" # token = "" # role_arn = "" # web_identity_token_file = "" #
+  role_session_name = "" # profile = "" # shared_credential_file = ""
```
Comment (powersj) [inline plugins/inputs/cloudwatch/README.md:34]: Was this auto-generated fixed? as the numbers should probably be kept on different lines and the `##` are already there

File: plugins/inputs/cloudwatch/README.md [hunk 3]
```diff
@@ -4,60 +4,50 @@ This plugin will pull Metric Statistics from Amazon CloudWatch.
...
-  #
-  ## Requested CloudWatch aggregation Period (required - must be a multiple of 60s)
-  period = "5m"
+  # metrics are made available to the 1 minute period. Some are collected at # 3
+  minute, 5 minute, or larger intervals. See
+  https://aws.amazon.com/cloudwatch/faqs/#monitoring.  # Note that if a period
+  is configured that is smaller than the minimum for a # particular metric, that
+  metric will not be returned by the Cloudwatch API # and will not be collected
+  by Telegraf.  # ## Requested CloudWatch aggregation Period (required - must be
+  a multiple of 60s) period = "5m"
```
Comment (powersj) [inline plugins/inputs/cloudwatch/README.md:50]: Same with the items above re: different lines

**Review by powersj — COMMENTED** [review]
Awesome! Only some questions about cloudwatch's README

File: plugins/inputs/cloudwatch/README.md (hunk 1 above)
Comment (reimda) [reply plugins/inputs/cloudwatch/README.md]: I rebased this PR to master and this change to config.conf wasn't needed.

**Pull request merged**